tools/build_rules/prelude_bazel

```diff
@@ -0,0 +1,11 @@
+load(
+    "@io_bazel_rules_pex//pex:pex_rules.bzl",
+    "pex_binary",
+    "pex_library",
+    "pex_test",
+    "pex_pytest",
+)
+load(
+    "@bazel_tools//tools/build_defs/pkg:pkg.bzl",
```

Comment:
I believe `@bazel_tools` is defined internally by Bazel; it replaces the former implicit merging of `//tools` into the user workspace.